Adding Warning section about Secure by Default

magilbe opened the pull request:
Adding the following warning section so that customers know that making this upgrade does then require an NSG associated to the Interface: 

Be aware that Standard SKU IP Addresses are considered Secure by default and will be closed to all internal traffic. It is recommended to apply an NSG to the Subnet or Network Interface that the Public IP address is assoicated in order to prevent unexpected downtime, see Public IP addresses in Azure

magilbe committed:
Adding Warning section about Secure by Default

Adding the following warning section so that customers know that making this upgrade does then require an NSG associated to the Interface: 

Be aware that Standard SKU IP Addresses are considered Secure by default and will be closed to all internal traffic. It is recommended to apply an NSG to the Subnet or Network Interface that the Public IP address is assoicated in order to prevent unexpected downtime, see Public IP addresses in Azure

v-dirichards commented on articles/virtual-network/ip-services/public-ip-upgrade.md:
```diff
@@ -20,6 +20,9 @@ Azure public IP addresses are created with a SKU, either Basic or Standard. The
 In this article, you learn how to upgrade a static Basic SKU public IP address to Standard SKU in the Azure portal, Azure CLI, or Azure PowerShell.
 
+> [!WARNING]
+> Be aware that Standard SKU IP Addresses are considered Secure by default and will be closed to all internal traffic. It is recommended to apply an NSG to the Subnet or Network Interface that the Public IP address is assoicated in order to prevent unexpected downtime, see [Public IP addresses in Azure](https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/public-ip-addresses#sku)
```
```suggestion
> Be aware that Standard SKU IP Addresses are considered Secure by default and will be closed to all internal traffic. It is recommended to apply an NSG to the Subnet or Network Interface that the Public IP address is assoicated in order to prevent unexpected downtime, see [Public IP addresses in Azure](https://learn.microsoft.com/azure/virtual-network/ip-services/public-ip-addresses#sku)
```
fixing build error

mbender-ms commented on articles/virtual-network/ip-services/public-ip-upgrade.md:
```diff
@@ -20,6 +20,9 @@ Azure public IP addresses are created with a SKU, either Basic or Standard. The
 
 In this article, you learn how to upgrade a static Basic SKU public IP address to Standard SKU in the Azure portal, Azure CLI, or Azure PowerShell.
 
+> [!WARNING]
```
```suggestion
> [!IMPORTANT]
```
Please change the call-out to use IMPORTANT vs. WARNING.

magilbe committed:
Update articles/virtual-network/ip-services/public-ip-upgrade.md

changing tag from warning to important

Co-authored-by: Michael Bender <102542398+mbender-ms@users.noreply.github.com>

magilbe committed:
Update articles/virtual-network/ip-services/public-ip-upgrade.md

changing URL to apply to all regions

Co-authored-by: Diana Richards <v-dirichards@microsoft.com>

mbender-ms commented on articles/virtual-network/ip-services/public-ip-upgrade.md:
```diff
@@ -20,6 +20,9 @@ Azure public IP addresses are created with a SKU, either Basic or Standard. The
 In this article, you learn how to upgrade a static Basic SKU public IP address to Standard SKU in the Azure portal, Azure CLI, or Azure PowerShell.
 
+> [!WARNING]
+> Be aware that Standard SKU IP Addresses are considered Secure by default and will be closed to all internal traffic. It is recommended to apply an NSG to the Subnet or Network Interface that the Public IP address is assoicated in order to prevent unexpected downtime, see [Public IP addresses in Azure](https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/public-ip-addresses#sku)
```
```suggestion
> Standard SKU IP Addresses are deployed using the [**Secure by default**](public-ip-addresses.md#sku) model. By default, IP addresses will be closed to all internal traffic when used as a front-end. To allow traffic and prevent unexpected downtime, apply a [network security group (NSG)](../../virtual-network/network-security-groups-overview.md#network-security-groups) to public IP address' associated Network Interface (NIC) or subnet.
```

magilbe replied:
@mbender-ms I made the changes to the pull request. Please let me know if there is anything else I had missed. Thanks!

mbender-ms reviewed (changes requested):
Thanks for the suggestions @magilbe.

Please update your PR with the suggested changes, then it can be merged.
- Updated callout title
- Change URLs to path-based
- Re-word callout and added link to NSG information

magilbe committed:
Changing important section to use Relative Link instead of Absolute

Changed the link to "Public IP addresses in Azure" to use the relative link: ./public-ip-addresses.md#sku

magilbe committed:
Updated verbiage of Secure by default messaging

Updated verbiage of Secure by default messaging by adding a link for applying a Network Security Group
